css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,18 @@
+button {
+  font-size: 20px;
+}
+
+button:hover {
+  background-color: black;
+  color: white;
+  cursor: pointer;
+}
+
+th {
+  background-color: black;
+  color: white;
+}
+
+tr:nth-child(2n+1) {
```

Comment:
I think you were doing this because of the `tr` element in the `thead` element?

Instead try using a more specific selector that targets `tr` elements within a `tbody` element.

That way, you can just use the `odd` keyword instead of the function in your pseudo selector :)

Reply:
Do you think I need to be more specific, like adding classes into table?

Reply:
That would be the best! But for now, I will approve and close this one! Great job!

Reply:
Got it!